docs/sources/setup-grafana/installation/docker/index.md

```diff
@@ -8,252 +8,316 @@ weight: 200
+   For example:
 
-The [Grafana Image Renderer plugin]({{< relref "../../image-rendering/#grafana-image-renderer-plugin" >}}) does not currently work if it is installed in a Grafana Docker image. You can build a custom Docker image by using the `GF_INSTALL_IMAGE_RENDERER_PLUGIN` build argument. This installs additional dependencies needed for the Grafana Image Renderer plugin to run.
+   ```bash
+    version: "3.8"
+    services:
+      grafana:
+      image: grafana/grafana-enterprise
+      container_name: grafana
+      restart: unless-stopped
+      ports:
+        - '3000:3000'
+   ```
 
-Example of how to build and run:
+1. To run `docker-compose.yaml`, run the following command:
 
-```bash
-cd packaging/docker/custom
-docker build \
-  --build-arg "GRAFANA_VERSION=latest" \
-  --build-arg "GF_INSTALL_IMAGE_RENDERER_PLUGIN=true" \
-  -t grafana-custom .
+   ```bash
+   # first go into the directory where you have created this docker-compose.yaml file
+   cd /path/to/docker-compose-folder 
 
-docker run -d -p 3000:3000 --name=grafana grafana-custom
-```
+   # then start the grafana container 
+   docker compose up -d docker-compose.yaml
+   ```
 
-## Migrate from previous Docker containers versions
+   Where:
 
-This section contains important information if you want to migrate from previous Grafana container versions to a more current one.
+   d = detached mode \
+   up = to bring the container up and running
 
-### Migrate to v7.3 or later
+To determine that Grafana is running, open a browser window and type `IP_ADDRESS:3000`. The sign in screen should appear.
 
-The Grafana Docker image runs with the `root` group (id 0) instead of the `grafana` group (id 472), for better compatibility with OpenShift. If you extend the official Docker image you may need to change your scripts to use the `root` group instead of `grafana`.
+### Stop the Grafana container
 
-### Migrate to v6.5 or later
+To stop the Grafana container, run the following command:
 
-Grafana Docker image now comes in two variants, one [Alpine](http://alpinelinux.org) based and one [Ubuntu](https://ubuntu.com/) based, see [Image Variants](#image-variants) for details.
+```bash
+docker compose down
+```
 
-### Migrate to v6.4 or later
+> **Note:** For more information about using Docker Compose commands, refer to [docker compose](https://docs.docker.com/engine/reference/commandline/compose/).
 
-Grafana Docker image was changed to be based on [Alpine](http://alpinelinux.org) instead of [Ubuntu](https://ubuntu.com/).
+### Save your Grafana data
 
-### Migrate to v5.1 or later
+When you use Docker containers, the data inside them is temporary by default. This means that if you don't specify where to store the information, all the Grafana data will be lost when you stop the Docker container. To avoid losing your data, you can set up [persistent storage](https://docs.docker.com/storage/volumes/) or [bind mounts](https://docs.docker.com/storage/bind-mounts/) for your container.
```

Comment:
```suggestion
When you use Docker containers, their data is temporary by default. If you don't specify where to store the information, all the Grafana data will be lost when you stop the Docker container. To avoid losing your data, you can set up [persistent storage](https://docs.docker.com/storage/volumes/) or [bind mounts](https://docs.docker.com/storage/bind-mounts/) for your container.
```